lib/ui/profile/interactor/profile_bloc.dart

```diff
@@ -56,15 +63,17 @@ class ProfileBloc extends Bloc<ProfileEvent, ProfileState> {
       if (result.isValue) {
         emit(state.copyWith(pageState: PageState.success, profileData: result.asValue!.value));
       } else {
+        final error = result.asError!.error;
         // if loading fails, show saved user data.
         final profileData = ProfileData(
-          name: userData.userName,
+          name: userData.userName ?? '',

```

Comment:
i dont think this is wise. 

null is not the same as ''

Reply:
Right.. I guess the null would then go to the user interface.

Reply:
Will change - this was strange, because passing null here would cause a crash

The name field is optional in ProfileData, but the constructor had a required - it was weird because the compiler didn't catch this.

Reply:
Fixed and merging. Good catch, there was a much better fix just removing the 'required' from the parameter.